File: src/Python/901-1000/961.RepeatedNTimes.py

class Solution:
    def repeatedNTimes(self, A):
        """
        :type A: List[int]
        :rtype: int
        """
        from collections import Counter
        counteritems = Counter(A).items()
        for k, v in counteritems:
            if v > len(A) / 2 - 1:
                return k

if __name__ == '__main__':
    solution = Solution()
    print(solution.repeatedNTimes([1,2,3,3]))
    print(solution.repeatedNTimes([2,1,2,5,3,2]))
    print(solution.repeatedNTimes([5,1,5,2,5,3,5,4]))
else:
    pass

File: src/CSharp/901-1000/961.RepeatedNTimes.cs

using System;
using System.Collections.Generic;

namespace PeefyLeetCode.VerifyingAnAlienDictionary
{
public class Solution {
    public int RepeatedNTimes(int[] A) {
        HashSet<int> set = new HashSet<int>();
        for (int i = 0; i < A.Length; ++i) {
            if (set.Contains(A[i])) {
                return A[i];
            }
            else
                set.Add(A[i]);
        }
        return 0;
    }
}
}

